src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -125,7 +124,8 @@ private void initialize() {
                     */
                     if (entriesListView.getItems().size() == 1) {
                         selectAllNewEntries();
-                      }
+                    }
+                    totalItems.textProperty().setValue(String.valueOf(entriesListView.getItems().size()));
```

Comment:
It should be possible to bind the text directly to the size:
`totalItems.textProperty().bind(Bindings.size(viewModel.getEntries()))`, which can then be moved to line 90.

Reply:
It actually didn't work as you said, I just pushed some changes that were missing, I hope everything is ok.